platform/linux-generic/Makefile.am

```diff
@@ -373,3 +373,16 @@ __LIB__libodp_linux_la_LIBADD += $(TIMER_LIBS)
 if ODP_PKTIO_PCAP
 __LIB__libodp_linux_la_LIBADD += $(PCAP_LIBS)
 endif
+
+CHECK_GLOBALS_REGEX = " (_deprecated_)?_?odp_| miniz_| mz_| tdefl_| tinfl_| mp_hdlr_init_odp_pool_ops"
```

Comment:
The first rule could be written open for better readability:
_deprecated_odp_ | _odp_ | odp_